tests/test_tox_uv_venv.py

```diff
@@ -339,3 +339,19 @@ def test_uv_python_set(tox_project: ToxProjectCreator, monkeypatch: pytest.Monke
     })
     result = project.run("-vv")
     result.assert_success()
+
+
+def test_uv_pip_constraints(tox_project: ToxProjectCreator) -> None:
+    project = tox_project({
+        "tox.ini": """[testenv]
+npackage=skip
+deps=setuptools
+setenv=
+  PIP_CONSTRAINTS=/dev/null
+commands=python -c 'import setuptools'
+"""
```

Comment:
Indent to same level as the code. Also put `[testenv]` on line to, for easier read, tox will ignore the first empty line. npackage => package.

Reply:
That is cool, I did not know that you added dedent to it, clearly make it easier to read.